benchmarks/solkz/compositional_fields/solkz_compositional_fields.prm

```diff
@@ -0,0 +1,99 @@
+# A description of the SolKZ benchmark using compositional fields
+# for which a known solution is available. See the manual for
+# more information.
+
+set Additional shared libraries            = ./libsolkz_compositional_fields.so
+
+############### Global parameters
+
+set Dimension                              = 2
+
+set Start time                             = 0
+set End time                               = 0
+
+set Output directory                       = output
+
+set Pressure normalization                 = volume
+set Nonlinear solver scheme                = Stokes only
+
+############### Parameters describing the model
+
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X extent = 1
+    set Y extent = 1
+  end
+end
+
+subsection Model settings
+  set Prescribed velocity boundary indicators =
+  set Tangential velocity boundary indicators = left, right, bottom, top
+  set Zero velocity boundary indicators       =
+end
+
+subsection Material model
+  set Model name = SolKzCompositionalMaterial
+end
+
+subsection Gravity model
+  set Model name = vertical
+end
+
+############### Parameters describing the temperature field
+
+subsection Boundary temperature model
+  set Model name = box
+end
+
+subsection Initial temperature model
+  set Model name = perturbed box
+end
+
+############### Parameters describing the discretization
+
+subsection Discretization
+  set Stokes velocity polynomial degree       = 2
+  set Use locally conservative discretization = true
```

Comment:
Is there a reason you use discontinous pressures for the compositional benchmark but continuous pressures for the existing solkz benchmark? This will confuse people trying to run this benchmark. It also confused me to see different numbers of degrees of freedom in the results you posted in the pull request. It also makes me wonder if the difference in accuracy is due to this difference in the input file. Can you change this to `false` and still get similar results? Then please do this. Lets discuss what to do, if that is not the case.